helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -282,6 +282,11 @@ processors:
         - key: splunk.com/index
           tag_name: com.splunk.index
           from: pod
+        {{- if .Values.logsCollection.containers.useSplunkIncludeAnnotation }}
+        - key: splunk.com/include
+          tag_name: splunk.com/include
+          from: pod
+        {{- end }}
```

Comment:
Can we take `splunk.com/exclude` annotation under `else` clause to avoid unnecessary attributes manipulations?

Reply:
Btw why don't we take `splunk.com/include` from namespace? Just curious

Reply:
The feature request from feedback was only for pods. We could add namespace support in future if there is customer demand!

Reply:
Do you see any issue with having consistent behavior for the filtering from the start?

If you agree, we could even add a helper for the filtering attribute that would return `splunk.com/include` or `splunk.com/exclude` depending on the configuration. Then all the places where `splunk.com/exclude` is used can be just replaced with `{{ include "splunk-otel-collector.filterAttr" . }}`. It'll remove unnecessary attribute manipulations which is always good for performance

Reply:
I think we should probably do that later during the refactor.

Reply:
This doesn't seem to be a big change, not sure why it has to introduce any tech debt